app/main.py

```diff
@@ -1,5 +1,76 @@
+from typing import Optional, List
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: Optional[List[int]] = None
+                 ) -> None:
```

Comment:
Start every argument from a new line

Reply:
After def __init__(...) -> None: - a blank line is NOT needed, since the if statement is the first line of code in the method.
PEP 8 doesn't recommend adding blank lines at the beginning of functions/methods, only between logically different blocks within them.

Reply:
You can use 2 code styles. If you start `self` param at the same line with `(` - you should end your last param with `)` at the same line.

Example:
```suggestion
    def __init__(self,
                 name: str,
                 weight: int,
                 coords: Optional[List[int]] = None) -> None:
```

Example:
```suggestion
    def __init__(
        self,
        name: str,
        weight: int,
        coords: Optional[List[int]] = None
    ) -> None:
```